includes/admin/upgrades/v3/class-data-migrator.php

```diff
@@ -1214,6 +1234,55 @@ public static function tax_rates( $data = null ) {
 		edd_add_adjustment( $adjustment_data );
 	}
 
+	/**
+	 * Normalizes and backfills legacy payment cart data.
+	 *
+	 * @since 3.0.0
+	 *
+	 * @param array $cart_details Cart details.
+	 * @return array
```

Comment:
Technically this could also return a string--I think that is the other possible type of `$cart_details` that could be passed to the function (same is true for the parameter I guess).